cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cache/RedisCache.java

```diff
@@ -56,11 +57,13 @@
     private final String prefix;
     private final JedisSource source;
     private final ObjectMapper objectMapper;
+    private final int maxMsetSize;
 
-    public RedisCache(String prefix, JedisSource source, ObjectMapper objectMapper) {
+    public RedisCache(String prefix, JedisSource source, ObjectMapper objectMapper, int maxMsetSize) {
         this.prefix = prefix;
         this.source = source;
         this.objectMapper = objectMapper.disable(SerializationFeature.WRITE_NULL_MAP_VALUES);
+        this.maxMsetSize = maxMsetSize;
```

Comment:
I see below where it is correcting a potentially bad value, I'd just roll that up into this constructor, or throw an exception here if it is invalid